bayesian_benchmarks/data.py

```diff
@@ -753,6 +755,173 @@ class C(Classification):
         name, N, D, K = name, N, D, K
 
 
+path_to_mujoco_dataset = '/home/felix/Desktop/policy_destination'
+policy_checkpoints = [str(i * 100000) for i in range(11)]
+evaluation_suffix = 'sac_policy.eval'
+evaluations = 10
+
+
+class MujocoSoftActorCriticDataset(Dataset):
+    """
+    The dimensions of the observation space and the action space are given by
```

Comment:
Can you add a bit more information about the dataset

Reply:
Done